pre_commit_hooks/pretty_format_json.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+import sys
+
+import simplejson
+
+
+def pretty_format_json(argv=None):
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--autofix',
+        action='store_true',
+        dest='autofix',
+        help='Automatically fixes encountered not-pretty-formatted files'
+    )
+
+    parser.add_argument('filenames', nargs='*', help='Filenames to fix')
+    args = parser.parse_args(argv)
+
+    status = 0
+
+    for json_file in args.filenames:
+        try:
+            f = open(json_file, 'r')
+            contents = f.read()
+            f.close()
+
+            pretty_contents = simplejson.dumps(
+                simplejson.loads(contents),
+                sort_keys=True,
+                indent=4
```

Comment:
Would it be possible to make this configurable?

Our project is using 2 spaces since that's the output of most JSON tools, such as `jq`.